test_app/views.py

```diff
@@ -32,3 +35,15 @@ class EncryptionModelViewSet(TestAppViewSet):
 class RelatedFieldsTestModelViewSet(TestAppViewSet):
     queryset = RelatedFieldsTestModel.objects.all()  # needed for automatic basename from router
     serializer_class = serializers.RelatedFieldsTestModelSerializer
+
+
+# create api root view from the router
+@api_view(['GET'])
+def api_root(request, format=None):
+    return Response(
+        {
+            'organizations': reverse('organization-list', request=request, format=format),
+            'teams': reverse('team-list', request=request, format=format),
+            'users': reverse('user-list', request=request, format=format),
```

Comment:
Actually now I'm wondering if this might be _less_ useful than a 404, because in the event of a 404 it gives you a list of the patterns and names for them. What might be _best_ is to programatically loop over the URLs and auto-generate this. Even if it's a little gnarly, that's fine for test_app.

Reply:
yeah would be nice to just collate these endpoints automatically, let me look into it..